deploy/yaml/kmesh.yaml

```diff
@@ -54,7 +54,7 @@ spec:
                   path: istio-token
       containers:
         - name: kmesh
-          image: ghcr.io/kmesh-net/kmesh:latest
```

Comment:
It's better to open a new folder for e2e's yaml

Reply:
Good catch, I should find a way to configure HUB of the image

Reply:
How do you install, if using helm we can customize the image

Reply:
Now I apply yamls directly for convenience and I will use helm later.